vizier/backend/src/info/vizierdb/api/GetArtifact.scala

```diff
@@ -42,6 +43,19 @@ object GetArtifact
         expecting.isEmpty || expecting.get.equals(artifact.t)
       }
 
+  /*
+   This is where the Original Vizier data profiler method will be placed 
+  */
+  // Never run the profiler unless True
+  // run the profiler before the return {} statement
+  // make sure when you run the profiler then you set all the stuff using updateDatasetProperty 
+  // i.e.:
+  /*
+      "is_profiler" : [profiler1, profiler2, profiler3, ...] <- verify this contains the profiler used
+      "column" : [..., ..., ...]
+      "count": [..., ..., ...]
+  */
```

Comment:
Please avoid committing development comments.  Comments are good, but they should support the code, rather than being a todo list.